examples/fabric-bridge-app/linux/RpcClient.cpp

```diff
@@ -107,9 +107,13 @@ void RpcCompletedWithEmptyResponse(const pw_protobuf_Empty & response, pw::Statu
 
 } // namespace
 
-CHIP_ERROR InitRpcClient(uint16_t rpcServerPort)
+void SetRpcClientPort(uint16_t port)
```

Comment:
There is no Client Port concept in RPC, we need to rename it to SetRpcRemoteServerPort